### ISSUE TITLE
Expand $check command to support opt-out checks for specified users

### ISSUE THREAD

**Issue by zneix** [Supinic/supibot-package-manager#46]
I would like a new type of check, it could be named `optout` or something similar.

Example usage:
`$check optout fourtf notify` => `This user has opted out from remind (notify) command`

Not sure how would this work, but I could possibly see something like "total optouts" of a specified user, like so:
`$check optouts fourtf` => `This user has opted out from: remind, tuck and rl`
`$check optouts` => `You haven't opted out from anything`

**Comment by Supinic** [comment]
Yes, this could definitely be an expansion of the `check` command. Opt-outs are public, since they can be both tested with the given user-command combo; plus, they're publically listed on each command's detail.

However, I'd like to note that blocks do not behave this way, as they're user-user specific. So they should not be included.

@zneix Would you be interested in creating a PR for this issue?
